docs/admin/auth/methods.rst

```diff
@@ -132,10 +132,15 @@ Token must contain the following claims:
 
 ``username`` - user name in a third party app.
 
-``iss``, ``username`` and ``aud`` values must match the values created by
-``CREATE USER`` statement. If ``aud`` has not been defined on the
-``CREATE USER`` statement, the cluster id is used and must match the token's
-``aud`` value. See :ref:`create-user-jwt` for details.
+The token's ``iss``, ``username`` and ``aud`` values must match the
+corresponding values within CrateDB.
+CrateDB uses either the values defined on a user level via ``CREATE USER`` or if
+absent the global configuration.
+
+``aud`` defaults to the cluster id if absent from both the user definition
+and the global configuration.
+
+See :ref:`create-user-jwt` and :ref:`jwt_defaults` for details.
```

Comment:
Can you explain a bit more what happens. If I have a JWT token that matches the global setting, then what? Do I become the _crate admin user?

Reply:
If you have a JWT token that matches the global setting, you will be authenticated as a user that specified in token’s “username” claim.

And even knowing global setting for issuer doesn’t mean we put “crate” to the username and auth as a superuser. Or say we know a name of  “some_admin_user” and cannot use it in the token.

Lookup will find “candidate CrateDB user to auth”, claims will match but signature check would fail. We can generate some valid but fake token but it will be signed by some custom private key (because we don't know private keys of the side issuing tokens). https://security.stackexchange.com/questions/245258/why-cant-i-generate-my-own-jwt-to-fake-authentication